python/ray/serve/_private/logging_utils.py

```diff
@@ -203,32 +202,44 @@ def record_factory(*args, **kwargs):
     stream_handler.addFilter(log_to_stderr_filter)
     logger.addHandler(stream_handler)
 
-    logs_dir = get_serve_logs_dir()
-    os.makedirs(logs_dir, exist_ok=True)
-    if max_bytes is None:
+    if logging_config.enable_access_log:
```

Comment:
Seems like `enable_access_log` controls whether we write logs to log files under the directory `/tmp/ray/session_latest/logs/serve/`. However the description of `enable_access_log` in schema.py is "Whether to enable access logs for each request. Default to True."

Reply:
Updated the description.